### PR TITLE
JSON to support balance, paid in this month and first bonus stories.

### PR DESCRIPTION
**Pull request by jameshartt**
As discussed in Slack, here is a first go at the JSON for further HtS work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,17 @@ Response format:
   "user": {
     // user state, can be NotEnrolled, InvitedFirstTime, Invited or Enrolled. See <confluence>/display/NGC/Help+to+Save+User+States
     "state": "NotEnrolled"
+  },
+  // App based feature toggles
+  "balanceEnabled": true,
+  "paidInThisMonthEnabled": true,
+  "firstBonusEnabled": true,
+  // Account-specific Help to Save data
+  "account": {
```
Comment (frankoid) [inline README.md]: I reckon that perhaps this should go inside the `user` object (`user.account`), the idea behind the `user` object is that it should contain things that vary between different users.

`account` could be seen as seperate though I suppose...

File: README.md [hunk 2]
```diff
@@ -26,6 +26,17 @@ Response format:
...
+    "balance": 100.50,
+    "paidInThisMonth": 50.25,
```
Comment (frankoid) [inline README.md]: A value <= 50 would be more realistic

File: README.md [hunk 3]
```diff
@@ -26,6 +26,17 @@ Response format:
   "user": {
     // user state, can be NotEnrolled, InvitedFirstTime, Invited or Enrolled. See <confluence>/display/NGC/Help+to+Save+User+States
     "state": "NotEnrolled"
+  },
+  // App based feature toggles
```
Comment (frankoid) [inline README.md]: I'm intending that these feature toggles will apply to the server too, so for example if `balanceEnabled == false` then the server will not call NS&I to obtain the balance (when we build that) and will omit `account.balance` from the JSON.

File: README.md (hunk 2 above)
Comment (jameshartt) [reply README.md]: Yep, you're right.

File: README.md (hunk 1 above)
Comment (jameshartt) [reply README.md]: I thought so, but remember us having a brief chat about this and saying that we weren't too sure. I'll put it this way around. and see.

File: README.md (hunk 3 above)
Comment (jameshartt) [reply README.md]: Sure 😄

**Comment by jameshartt** [comment]
@frankoid addressed comments, please take another look.

**Pull request merged**